test/memory.cpp

```diff
@@ -893,3 +1058,55 @@ TEST(MemoryManagerApi, E2ETest) {
     ASSERT_EQ(payload->initializeCalledTimes, 1);
     ASSERT_EQ(payload->shutdownCalledTimes, af::getDeviceCount());
 }
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+TEST(Memory, AfAllocDeviceCPUC) {
```

Comment:
`AfAllocDeviceCPUC` what does `C` in the end mean ? for this and the following tests

Reply:
Uses the C api instead of the C++ api